samples/pom.xml

```diff
@@ -33,6 +33,24 @@
     <module>snippets</module>
   </modules>
 
+  <dependencies>
```

Comment:
Just for completion, as we already discussed this, the samples changes should be removed from this PR